test/bdd/fixtures/adapter-rest/docker-compose.yml

```diff
@@ -14,6 +14,9 @@ services:
       - ADAPTER_REST_HOST_URL=0.0.0.0:8070
       - ADAPTER_REST_TLS_CACERTS=/etc/tls/ec-cacert.pem
       - ADAPTER_REST_TLS_SYSTEMCERTPOOL=true
+      - ADAPTER_REST_MYSQL_DATASOURCE=mysql://root:secret@tcp(mysql:3306)/mysql
+      - ADAPTER_REST_OP_URL=http://PUT-SOMETHING-HERE.com
+      - ADAPTER_REST_STATIC_FILES=/usr/local/static/adapter-vue
```

Comment:
Is this property mandatory ? This is already bundled inside docker. Are we going to expect consumers to build their own VueJS project with all JS/CHAPI calls ? Now that i think about it, is adapter-vue really static ?

Reply:
I agree not needed

Reply:
it's part of the image now, deployer will not need to set it

Reply:
so should we hardcode the path to the static files in the Dockerfile or in the binary?

Reply:
> is adapter-vue really static ?

It better be. `adapter-rest` should be the brains of the body, with the frontend just a thin, dumb client. This eases support of flows that do not interact with the wallet.

Reply:
@fqutishat @rolsonquadras hardcoded the parameter in the `Dockerfile`, please check